ui/desktop/electron-app/package.json

```diff
@@ -35,11 +35,11 @@
     "which": "^4.0.0"
   },
   "devDependencies": {
-    "@electron-forge/cli": "^7.2.0",
-    "@electron-forge/maker-deb": "^7.2.0",
-    "@electron-forge/maker-dmg": "^7.2.0",
-    "@electron-forge/maker-squirrel": "^7.2.0",
-    "@electron-forge/maker-zip": "^7.2.0",
```

Comment:
this is not related to this security vulnerability but looking into it made me realize it was resolving to 7.2.0 when 7.4.0 is out so figured we might as well keep it updated. lemme know if you disagree and would like this to be removed.

Reply:
removed since this change was causing errors in [release pipeline](https://github.com/hashicorp/boundary-ui-releases/actions/runs/10587353806/job/29338119604)